utilities/avro-schema-generator/src/main/java/com/flipkart/aesop/avro/schemagenerator/data/MysqlToAvroMapper.java

```diff
@@ -112,7 +126,7 @@ private MysqlToAvroMapper(String avroType)
 	 * Gets the avro type.
 	 * @return the avro type
 	 */
-	public String getAvroType()
+	public Object getAvroType()
```

Comment:
Because now hashmap is also being returned, previously only returning string (like "long", "int" etc)